Makefile

```diff
@@ -191,3 +196,25 @@ olm-deploy-example: olm-deploy olm-deploy-example-storage-secret ## Deploy examp
 olm-undeploy: $(OPERATOR_SDK) ## Cleanup deployments of the operator bundle and the operator via OLM on an OpenShift cluster selected via KUBECONFIG.
 	$(OPERATOR_SDK) cleanup loki-operator
 	kubectl delete ns $(CLUSTER_LOGGING_NS)
+
+.PHONY: deploy-size-calculator
+ifeq ($(findstring openshift-logging,$(CALCULATOR_IMG)),openshift-logging)
+deploy-size-calculator: ## Deploy storage size calculator (OpenShift only!)
+	$(error Set variable REGISTRY_ORG to use a custom container registry org account for local development)
+else
+deploy-size-calculator:  $(KUSTOMIZE) ## Deploy storage size calculator (OpenShift only!)
+	kubectl apply -f config/overlays/openshift/size-calculator/cluster_monitoring_config.yaml
+	kubectl apply -f config/overlays/openshift/size-calculator/user_workload_monitoring_config.yaml
+	sh hack/deploy-prometheus-secret.sh
```

Comment:
Why not simply `./hack/deploy-prometheus-secret.sh`?

Reply:
That will error out as `permission denied`

Reply:
The you should give it execute perms?

Reply:
Done